yarn.lock

```diff
@@ -17839,7 +17859,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"typescript@npm:^5.9.3":
+"typescript@npm:^5.9.2, typescript@npm:^5.9.3":
```

Comment:
👀 sign that we need to bump a TS version for consistency.

Reply:
Fixed in [3c81fdc](https://github.com/PrairieLearn/PrairieLearn/pull/13013/commits/3c81fdc45cb99e2591565b5df73e42eb5370f6e0)